tlslite/recordlayer.py

```diff
@@ -717,7 +724,9 @@ def _decryptAndUnseal(self, recordType, buf):
                                                 plaintextLen//256,
                                                 plaintextLen%256])
         else:  # TLS 1.3
-            authData = bytearray(0)
+            # this is essentially a Record Layer header
+            authData = bytearray([recordType, 3, 3,
+                                  len(buf) // 256, len(buf) % 256])
```

Comment:
I understand that this is probably easier to recreate the record layer header, but wouldn't it be more future-proof to carry it over from the actual data received?

Reply:
this has the side-effect of enforcing them to those values, and I want to be strict on receiving

(on sending side I'm setting them to configured values as I want to be able to test this in other implementations)

Reply:
fixed: 58a705e